basicview.cpp

```diff
@@ -15,39 +15,74 @@ limitations under the License.
 */
 
 #include "basicview.h"
-
 #include <QVBoxLayout>
+#include <QFontDatabase>
+#include <QFont>
+#include <QDebug>
+#include <QToolTip>
+
+const QString startIcon = "\uf04b";
+const QString stopIcon = "\uf04d";
+const QString pauseIcon = "\uf04c";
+const QString unPauseIcon = "\ue22f";
+const QString deleteIcon = "\uf1f8";
+const QString reloadIcon = "\uf021";
 
+void BasicView::setFA(QFont font, QWidget *wid)
```

Comment:
This isn't setting FA anymore, it's just assigning the given font, would make sense to give it a better name.

Reply:
done